views/users/new.hbs

```diff
@@ -1,3 +1,5 @@
+<h1>Sign up to Acebook</h1>
+
 <form id="new-user-form" enctype="multipart/form-data" action="/users" method="post">
   <label for="username">Username: </label>
```

Comment:
Move line 3 and 4 to after line 12 line breaks

Should show 
email address, password on one line.
line breaks
username, profile picture
line break
submit

see sceenshot

![Screenshot 2022-04-22 at 17 32 22](https://user-images.githubusercontent.com/98812145/164757102-c0beb03d-3fd1-4153-b9c0-e28e96fce63e.png)